pkg/rtc/mediatrack.go

```diff
@@ -36,8 +34,6 @@ type MediaTrack struct {
 	audioLevelMu sync.RWMutex
 	audioLevel   *AudioLevel
 
-	connectionStats *connectionquality.ConnectionStats
```

Comment:
Moving all these in `sfu.WebRTCReceiver` as that is a better place for all this.